packages/core/src/odata-common/request/odata-request-config.ts

```diff
@@ -143,6 +148,10 @@ export abstract class ODataRequestConfig {
     });
   }
 
+  appendPath(path: string): void {
```

Comment:
Could we use a `...string` for this method so that you can add one path or multiple in one call.